andela/travel_tool_back
# #162303455 Outflow for travel readiness should departure dates instead

#### What does this PR do?
For travel-readiness, outflows should have Expected Departure Date, not Expected Arrival Date as it is for inflows.

#### Description of Task to be completed?

#### How should this be manually tested?
* Clone the repo: ```https://github.com/andela/travel_tool_back```
* cd into their ```travel_tool_back```.
* Checkout to the branch ```ch-modify-readiness-outflow-data-162303455```.
* Start up the server with yarn start:dev.
* Login to the app on the front end.
* Make requests. Make sure to use your location as the origin. (As a travel Admin, this will enable you see them as outflows when they have been approved).
* Approve these requests.
* With a valid token, make a GET request with this route ```http://localhost:5000/api/v1/analytics/readiness?type=file&travelFlow=outflow```
* You should see ```Expected Departure Date``` instead of ```Expected Arrival Date``` (```Expected Arrival Date``` should be for inflows only).

#### Any background context you want to provide?
You should be logged in as a Travel Admin to view the dashboard.

#### What are the relevant pivotal tracker stories?
[162303455](https://www.pivotaltracker.com/story/show/162303455)

#### Screenshots (if appropriate)
<img width="1292" alt="screen shot 2018-11-29 at 2 47 35 pm" src="https://user-images.githubusercontent.com/29521319/49226005-c2704c80-f3e5-11e8-8b76-efc2079dfd88.png">

#### Questions:
